build.gradle

```diff
@@ -70,15 +55,6 @@ allprojects {
         jvmArgs '-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8188'
       }
     }
-
-    group = "com.netflix.spinnaker.igor"
-
-    //c&p this because NetflixOss reverts it to 1.7 and ends up getting applied last..
-    project.plugins.withType(JavaBasePlugin) {
-        JavaPluginConvention convention = project.convention.getPlugin(JavaPluginConvention)
-        convention.sourceCompatibility = JavaVersion.VERSION_1_8
-        convention.targetCompatibility = JavaVersion.VERSION_1_8
-    }
 }
 
 defaultTasks 'run'
```

Comment:
I just love how the bom stuff cleans up the main gradle file. 🥓

Reply:
indeed, is niiice